core/controllers/release_coordinator.py

```diff
@@ -246,9 +246,14 @@ def get(self) -> None:
         feature_flags_dict = []
         for feature_flag in feature_flags:
             feature_flags_dict.append(feature_flag.to_dict())
+        user_groups = user_services.get_all_user_groups()
+        user_groups_dict_list = []
```

Comment:
This name is a bit clumsy. Perhaps user_group_dicts

Reply:
Got it, updated.